addons/casings/functions/fnc_createCasing.sqf

```diff
@@ -0,0 +1,69 @@
+#include "script_component.hpp"
+/*
+ * Author: esteldunedain / Cyruz / Diwako
+ * Produces a casing matching the fired rifles caliber on the ground around the unit
+ *
+ * Arguments:
+ * 0: unit - Object the event handler is assigned to <OBJECT>
+ * 1: weapon - Fired weapon <STRING>
+ * 2: ammo - Ammo used <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, currentWeapon player, "","", player ammo currentWeapon player] call ace_casings_fnc_createCase
+ *
+ * Public: No
+ */
+
+params ["_unit", "_weapon", "", "", "_ammo"];
+
+if (!isNull objectParent _unit || { _weapon in ["Throw","Put"] } ) exitWith {};
```

Comment:
Excluding Throw and Put here seems like an overall loss. The casing would be "" anyway for these. I don't see why they are special compared to, e.g. a launcher or an ugl.